extendeddatacrossword_test.go

```diff
@@ -344,3 +345,143 @@ func createTestEds(codec Codec, shareSize int) *ExtendedDataSquare {
 
 	return eds
 }
+
+func TestCorruptedEdsReturnsErrByzantineData_UorderedShares(t *testing.T) {
+	shareSize := 64
```

Comment:
[optional]

```suggestion
	shareSize := 512
```

motivation: https://github.com/celestiaorg/rsmt2d/issues/237